gradio/chat_interface.py

```diff
@@ -430,11 +436,16 @@ async def api_fn(message, history, *args, **kwargs):
 
     def _clear_and_save_textbox(
         self, message: str | dict
-    ) -> tuple[str | dict, str | MultimodalData]:
+    ) -> tuple[Textbox | MultimodalTextbox, str | MultimodalData]:
+        placeholder = message if isinstance(message, str) else message.get("text", "")
         if self.multimodal:
-            return {"text": "", "files": []}, MultimodalData(**cast(dict, message))
+            return MultimodalTextbox(
+                {"text": "", "files": []}, interactive=False, placeholder=placeholder
+            ), MultimodalData(**cast(dict, message))
         else:
-            return "", cast(str, message)
+            return Textbox("", interactive=False, placeholder=placeholder), cast(
```

Comment:
I don't really like the placeholder because we've already cleared the textbox and the user message is displayed in the chatbot.

Minor point though.

Reply:
Yeah okay I can just get rid of it, no strong feelings on my part